stacks/spica/src/passport/pages/api-key-add/api-key-add.component.scss

```diff
@@ -0,0 +1,21 @@
+:host {
+  mat-card {
+    margin: 30px;
+    overflow: hidden;
+    padding: 25px;
+  }
+  mat-form-field {
+    width: 100%;
+    margin-top: 15px;
+  }
+  .resource {
+    padding-left: 30px;
+  }
+  .resource-input {
+    padding-left: 5px;
+  }
```

Comment:
Change class names to domain-specific ones. Even you may remove them cause I couldn't see where they were used.